src/routes/user/index.js

```diff
@@ -28,8 +28,12 @@ export async function userRoutes(server) {
       const response = createSuccessfulResponse("post", req.body, token);
       res.code(HTTP_CREATED).send(response);
     } catch (err) {
```

Comment:
We probably want to see if we catch a query error exception if it is it's just an internal server error as the message. The actual code we can see what error postgresql gives us and pass that error code to the error and use that.